druid-shell/src/platform/windows/window.rs

```diff
@@ -255,6 +271,9 @@ impl Drop for HCursor {
 /// Message indicating there are idle tasks to run.
 const DS_RUN_IDLE: UINT = WM_USER;
 
+/// Transparent bg clearing color
+const TRANSPARENT: Color = Color::rgba8(0, 0, 0, 0);
```

Comment:
Let's add a FIXME here to use the `Color::TRANSPARENT` const in piet whenever we do the next release there.